_rules/link-non-empty-accessible-name-c487ae.md

```diff
@@ -185,6 +185,14 @@ This `area` element has a [semantic role][] of `link` and an [accessible name][]
 </map>
 ```
 
+#### Passed Example 11
+
+This `span` element has an [inheriting semantic role][] of `link` and an [accessible name] given by its child text node.
+
+```html
+See [<span onclick="location.href='https://act-rules.github.io/'" role="doc-biblioref">ACT rules</span>]
```

Comment:
I'm a little hesitant to say this passes SC 4.1.2. This wouldn't be accessibility supported. Perhaps what we need to do is put these on actual anchor elements?